applications/plugins/CGALPlugin/src/CGALPlugin/CylinderMesh.inl

```diff
@@ -508,21 +508,21 @@ void CylinderMesh<DataTypes>::draw(const sofa::core::visual::VisualParams* vpara
         points.resize(m_nbVertices);
         for (int i = 0; i < m_nbVertices; ++i)
             points[i] = coords[i];
-        vparams->drawTool()->drawPoints(points,8,helper::types::RGBAColor(0.0, 0.0, 1.0,1));
+        vparams->drawTool()->drawPoints(points,8,type::RGBAColor(0.0, 0.0, 1.0,1));
```

Comment:
(just a comment) next time, if the color is obvious, replace it with the named function, like
`type::RGBAColor::green()` in this case

Reply:
Thanks,
I see and often use the new handy functions for colors
Since this was only a fix, I did not bother to change the source beyond compilation